README.md

```diff
@@ -11,86 +11,131 @@
+
+**Output**: All formulas for the chosen source table will be generated in a `select` statement. This output differs from Option 1 only in the format of compiled code. 
+
+> This option makes use of the `fivetran_formula` lookup table, which requires the package to combine fields' formulas into a SQL query for each source table. This option does not support double-nested formulas and may be incomplete compared to Option #1.
+
+### Option 3: Generate only specified formula fields using package-made query
+
+If you would like your model to generate only a specified subset of your formula fields related to your source table then you will need to: 
+1. Create a new file in your models folder and name it `your_table_name_here`.sql (e.g. `customer.sql`; this is not necessary but recommended as best practice). 
+2. Add the below snippet calling the `sfdc_formula_view` macro into the file and:
+    - Update the `source_table` argument to be the source table name for which you are generating the model (e.g. `customer`).
+    - Update the `fields_to_include` argument to contain all the fields from your source that you would like to be included in the final output. Be sure that the field(s) you would like to include are enclosed within brackets as an array (ie. `[]`)
 ```sql
 {{ salesforce_formula_utils.sfdc_formula_view(
     source_table='your_source_table_name_here', 
     fields_to_include=['i_want_this_field','also_this_one','maybe_a_third_as_well','lets_add_more'],
     full_statement_version=false) 
 }}
 ```
-### Formula Fields that Reference Other Formula Fields
-This macro has been created to allow for two degrees of formula field reference. For example:
+
+**Output**: Formulas provided in the `fields_to_include` variable will be generated in a `select` statement for the chosen source table.
+
+> This option makes use of the `fivetran_formula` lookup table, which requires the package to combine fields' formulas into a SQL query for each source table. This option does not support double-nested formulas.
+
+### Automate model creation
+If you have multiple models you need to create, you can also Leverage the [sfdc_formula_model_automation](https://github.com/fivetran/dbt_salesforce_formula_utils/blob/main/sfdc_formula_model_automation.sh) script within this project to automatically create models locally via the command line. Below is an example command to copy and edit.
+
+```bash
+source dbt_modules/salesforce_formula_utils/sfdc_formula_model_automation.sh "../path/to/directory" "desired_table_1,desired_table_2,desired_table_infinity"
+```
+
+By default, these models will run Option #1.
+
+## Step 5: Exclude problematic formula fields
+The `sfdc_formula_view` macro has been created to allow for two degrees of formula field reference. For example:
```

Comment:
I think two degrees makes sense, but I also noticed we referenced "double nested" a lot and I think it would make sense to use that instead here. I think the below are great examples of what a double nested formula would be and we could take advantage of this section to explain what "double nested" means for the reader.

Reply:
Based on the above `README`..

```suggestion
The `sfdc_formula_view` macro has been created to support double-nested formula field references for most cases using Option #1, however, never if using Options #2 or #3. For example:
```